.github/integration/setup/posix/30_posix_permissions.sh

```diff
@@ -3,3 +3,5 @@
 docker run --rm -v dev_utils_archive:/foo alpine sh -c "chmod 777 /foo"
 docker run --rm -v dev_utils_backup:/foo alpine sh -c "chmod 777 /foo"
 
+# we need to have permissions to delete the file
+sudo chmod -R 777 dev_utils/
```

Comment:
This will work in the CI tests, but it will make it tricky if someone runs these tests locally.

Maybe use a docker volume for the posixs test instead or the chance is that someone will run the test and commit the results i.e. deleting the test files from the repo.

Reply:
ok, I will adjust